### ISSUE TITLE
Official docker images for App and DB

### ISSUE THREAD

**Issue by Breee** [mattermost/mattermost-docker#513]
Currently i'm not able to find official docker images for the mattermost app and db.

It would be nice if there would be an automatic build and push to a registry (i.e. dockerhub).

If we want to use dockerhub as registry, we could also enable autobuilds for each new release.  
Else we could write a CircleCI pipeline.

**Comment by gpopesc** [comment]
yes, I have the same question. I wonder if I manually change the version make any difference.
In app Dockerfile is still ENV MM_VERSION=5.31.0

https://github.com/mattermost/mattermost-docker/blob/master/app/Dockerfile

**Comment by jwillmer** [comment]
I found and use the following images:

```
mattermost/mattermost-prod-db:v5.31.0
mattermost/mattermost-team-edition:5.31.0
```
